man/tmpfiles.d.xml

```diff
@@ -647,13 +647,13 @@ w- /proc/sys/vm/swappiness - - - - 10</programlisting></para>
 # an hour ago in "/tmp/foo/bar", are subject to time-based cleanup.
 d /tmp/foo/bar - - - - bmA:1h -</programlisting></para>
 
-      <para>Note that while the aging algorithm is run a 'shared' BSD file lock (see <citerefentry
+      <para>Note that while the aging algorithm is run an exclusive BSD file lock (see <citerefentry
       project='man-pages'><refentrytitle>flock</refentrytitle><manvolnum>2</manvolnum></citerefentry>) is
-      taken on each directory the algorithm descends into (and each directory below that, and so on). If the
-      aging algorithm finds a lock is already taken on some directory, it (and everything below it) is
-      skipped. Applications may use this to temporarily exclude certain directory subtrees from the aging
-      algorithm: the applications can take a BSD file lock themselves, and as long as they keep it aging of
-      the directory and everything below it is disabled.</para>
+      taken on each directory/file the algorithm decides to remove. If the aging algorithm finds a lock is
+      already taken on some directory/file, it (and everything below it) is skipped. Applications may use
+      this to temporarily exclude certain directory subtrees from the aging algorithm: the applications can
+      take a BSD file lock themselves, and as long as they keep it aging of the directory/file and everything
```

Comment:
maybe emphasize that it is up to the apps to decide whether to use a shared or exclusive lock, whatever fits their purpose better.

i.e. when deleting we'll take an exclusive on, but the apps get free reign, and can pick either. I think this deserves mention.